triveni-codev/GBWebApp-dev
# Justin_Outputs-Thomson-Tree-View-Hierarchy-Listing-In-View-Selected-Transport_1203763676971231_v2

ASANA tickets:
https://app.asana.com/0/1203637575337688/1203763676971231/f
https://app.asana.com/0/1203637575337688/1203763676971237/f
https://app.asana.com/0/1203637575337688/1203763676971235/f
https://app.asana.com/0/1203637575337688/1203763676971233/f

**Description:** created a function which will display the data of the selected transport in an output - **Thomson**, **Ericsson AVP**, **Ericsson MX5600**, **Ericsson MX8400** types

- amended treegrid UI view
- amended the function call for manipulating the Transport data
- added some ej2-syncfusion-treegrid libraries
- added some icons and displayed it in the treegrid ~ dynamically

**TO TEST:**
- Create or Edit an Output (using all output types mentioned in the description above) via angular or serverapp
- Associate a Transport in the Output
- Conduct a play-by-play edit in the Transport data (add Programs, Transport Stream, etc)
- Edit the newly added Output (with Transport) and click VIEW transport
- Check if the Treegrid view is correct and same with the serverapp

Note: CRUD of OUTPUT is an out-of-scope of this ticket, please disregard bugs you will encounter regarding this.

cc: @triveni-codev
